### PR TITLE
[IMP] mail: remove extra * and fix type definition

### PR DESCRIPTION
**Pull request by phenix-factory**

### CODE REVIEW

File: addons/mail/static/src/new/activity/activity_button.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/** @odoo-module **/
+/** @odoo-module */
```
Comment (seb-odoo) [inline addons/mail/static/src/new/activity/activity_button.js]: should have single * at the start of module too
(double * is for js doc, this is not a js doc)

Comment (phenix-factory) [reply addons/mail/static/src/new/activity/activity_button.js]: Done

File: addons/mail/static/src/new/views/fields/list_activity/list_activity.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class ListActivity extends Component {
             return this.props.record.data.activity_summary;
         }
         if (this.props.record.data.activity_type_id) {
-            return this.props.record.data.activity_type_id[1 /** display_name **/];
+            return this.props.record.data.activity_type_id[1 /** display_name */];
```
Comment (seb-odoo) [inline addons/mail/static/src/new/views/fields/list_activity/list_activity.js:19]: this is not a js doc either, could be single *

**Review by seb-odoo — APPROVED** [review]

File: addons/mail/static/src/new/views/fields/list_activity/list_activity.js (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/src/new/views/fields/list_activity/list_activity.js:19]: I'll merge, you can do change in a separate PR.

Comment (phenix-factory) [reply addons/mail/static/src/new/views/fields/list_activity/list_activity.js:19]: https://github.com/odoo-dev/odoo/pull/1541

**Pull request merged**